src/args.rs

```diff
@@ -1448,7 +1448,7 @@ impl ArgMatches {
         } else {
             self.is_present("with-filename")
             || self.is_present("vimgrep")
-            || paths.len() > 1
+            || paths.len() > 0
```

Comment:
Thanks for working on this bug! But I don't think this is right. `rg foo file` should not be printing the file path by default.